### PR TITLE
[fix] add test cases and fix `is_content_type_textual`

### PR DESCRIPTION
**Pull request by si3nloong**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [x] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

Nothing much, just add more test cases

### CODE REVIEW

**Comment by Rich-Harris** [comment]
thank you! mind fixing the lint errors?

for reference, it's helpful if you don't delete the pull request template — it includes a checklist that includes things like linting, and it's useful for maintainers to be able to understand the scope of a PR (whether it requires a version bump, etc)

File: packages/kit/src/runtime/paths.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { set_paths, base, assets } from './paths.js';
+
+test('paths', () => {
```
Comment (benmccann) [inline packages/kit/src/runtime/paths.spec.js:5]: `set_paths` might be a clearer name

but I question if this is really a useful test. just testing that setting a variable sets it - there doesn't seem to be much room for error. and I would expect this is already covered by our other tests

File: packages/kit/src/utils/error.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { coalesce_to_error } from './error.js';
+
+test('coalesce_to_error', () => {
```
Comment (benmccann) [inline packages/kit/src/utils/error.spec.js:5]: this is at least a little more complicated than the last test, but still would think this functionality is likely already covered by an integration test and that there's little room to go wrong here. I and the other maintainers agree that we should rely more on unit tests than integration tests, but I think the units could be for more complicated functionality

**Comment by si3nloong** [comment]
@benmccann can you review again

File: packages/kit/src/runtime/paths.spec.js (hunk 1 above)
Comment (si3nloong) [reply packages/kit/src/runtime/paths.spec.js]: removed

File: packages/kit/src/utils/error.spec.js (hunk 2 above)
Comment (si3nloong) [reply packages/kit/src/utils/error.spec.js]: test removed

**Review by Rich-Harris — APPROVED** [review]

**Comment by Rich-Harris** [comment]
thanks!

**Pull request merged**